nikola/plugins/command/theme.py

```diff
@@ -263,6 +275,17 @@ def list_available(self, url):
             print(theme)
         return 0
 
+    def list_installed(self):
+        """List all installed themes."""
+        print("Installed Themes:")
+        print("-----------------")
+        themes = []
+        themes_dirs = self.site.themes_dirs + [resource_filename('nikola', os.path.join('data', 'themes'))]
+        for tdir in themes_dirs:
+            themes += [(i, os.path.join(tdir, i)) for i in os.listdir(tdir)]
```

Comment:
Shouldn't you filter themes by isdir() just in case there is some random file in themes/ or similar?